src/glib.ml

```diff
@@ -72,6 +72,8 @@ module Io = struct
   type id
   external channel_of_descr : Unix.file_descr -> channel
     = "ml_g_io_channel_unix_new"
+  external channel_of_descr_socket : Unix.file_descr -> channel
+    = "ml_g_io_channel_unix_new_socket"
```

Comment:
Shouldn't this rather be `ml_g_io_channel_win32_new_socket` ?
Note that the change is not visible in the interface, so this would not matter for CoqIDE.